docs/index.rst

```diff
@@ -27,10 +27,10 @@ no matter if they are small and simple, or large and complex.
 
 .. code:: python3
 
-   >>> from usim import each, run
+   >>> from usim import delay, run
    >>>
    >>> async def metronome(period: float, sound: str):
-   ...     async for now in each(delay=period):
+   ...     async for now in delay(value=period):
```

Comment:
See above comment, they are the same example.

Reply:
Will be fixed.

Reply:
I just opened a new ticket for you :) See #45